step-ca/provisioners.mdx

```diff
@@ -1027,6 +1030,96 @@ To get a client certificate for a hardware-bound private key on your YubiKey:
    }
    ```
 
+#### ACME for Wire messenger clients
+
+<Alert severity="warning">
+  <div> 
+    The challenge types for Wire clients must be explicitly enabled in an ACME provisioner.
+    Currently the configuration can only be done through editing the configuration file.
+  </div>
+</Alert>
+
+The `wire-oidc-01` and `wire-dpop-01` challenges enable `step-ca` to issue certificates to [Wire](https://wire.com) messenger clients.
+Wire clients make use of a user and a device identity, both of which are challenged and verified, after which the identities are combined into a single X.509 certificate.
+
+Here's an example of an ACME provisioner with Wire support enabled in `$(step path)/config/ca.json`:
+
+```json
+...
+{
+    "type": "ACME",
+    "name": "wire",
+    "claims": {
+        "maxTLSCertDuration": "24h",
+        "defaultTLSCertDuration": "16h"
```

Comment:
Wire identity certs are necessarily long lived because a received message might be decoded up to 28 days later. Wire recommends a Wire identity certificate is valid for 60-90 days. 

```suggestion
        "maxTLSCertDuration": "2160h",
        "defaultTLSCertDuration": "1440h"
```